Tools/Scripts/libraries/webkitbugspy/webkitbugspy/radar.py

```diff
@@ -437,6 +437,10 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
             did_change = True
             radar.sourceChanges = '\n'.join(source_changes)
 
+        if see_also:
+            sys.stderr.write('Radar does not support the see_also field at this time\n')
+            return None
```

Comment:
I guess, kinda. I mean we could accept links to other Radar issues, but maybe that's Future Work.